website/docs/r/lb_server_pool.html.markdown

```diff
@@ -0,0 +1,129 @@
+    ip_address      = "1.1.1.1"
+    port            = 8443
+    monitor_port    = 9000
+    weight          = 1
+    min_connections = 0
+    max_connections = 100
+  }
+
+  member {
+    condition       = "drain"
+    name            = "member2"
+    ip_address      = "2.2.2.2"
+    port            = 7000
+    monitor_port    = 4000
+    weight          = 2
+    min_connections = 6
+    max_connections = 8
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `edge_gateway` - (Required) The name of the edge gateway on which the server pool is to be created
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
+* `name` - (Required) Server Pool name
+* `description` - (Optional) Server Pool description
+* `algorithm` - (Required) Server Pool load balancing method. Can be one of `ip-hash`, `round-robin`, `uri`, `leastconn`, `url`, or `httpheader`
+* `algorithm_parameters` - (Optional) Valid only when `algorithm` is `httpheader` or `url`. The `httpheader` algorithm
+parameter has one option `headerName=<name>` while the `url` algorithm parameter has option `urlParam=<url>`. 
+* `enable_transparency` - (Optional) When transparency is `false` (default) backend servers see the IP address of the
+traffic source as the internal IP address of the load balancer. When it is `true` the source IP address is the actual IP
+address of the client and the edge gateway must be set as the default gateway to ensure that return packets go through
+the edge gateway. 
+* `monitor_id` - (Optional) `vcd_lb_service_monitor` resource `id` to attach to server pool for health check parameters
+* `member` - (Optional) A block to define server pool members. Multiple can be used. See [Member](#member) and 
+example for usage details.
+
+
+<a id="member"></a>
+## Member
+
+* `condition` - (Required) State of member in a pool. One of `enabled`, `disabled`, or `drain`
+* `name` - (Required) Member name
+* `ip_address` - (Required) Member IP address
+* `port` - (Required) The port at which the member is to receive traffic from the load balancer.
+* `monitor_port` - (Required) Monitor Port at which the member is to receive health monitor requests. **Note:** can
+be the same as `port`
+* `weight` - (Required) The proportion of traffic this member is to handle. Must be an integer in the range 1-256.
+* `min_connections` - (Optional) The maximum number of concurrent connections the member can handle. **Note:** when the
+number of incoming requests exceeds the maximum, requests are queued and the load balancer waits for a connection to be
+released. 
+* `max_connections` - (Optional) The minimum number of concurrent connections a member must always accept.
+
+## Attribute Reference
+
+The following attributes are exported on the base level of this resource:
```

Comment:
Sorry, but what do you mean? An ask to rephrase/add more context.

Reply:
I wanted to separate that this is at `root level` of object while members have their IDs in child blocks. Cleaned this up.